portfolio/src/main/webapp/script/comment-script.js

```diff
@@ -89,6 +90,17 @@ async function deleteData() {
   getData();
 }
 
+/**
+ * Fetches image upload URL from blobstore-upload-url servlet and sets
+ * URL as action in {@code comment-image-form}.
+ */
+async function getSetBlobURL() {
```

Comment:
Here too you want it to be written as getSetBlobUrl

Reply:
done